### PR TITLE
Add setup cli command.

### LINKED ISSUE #5669: create `kolibri setup` CLI command that just runs `initialize`
### Observed behavior

During setup of cloud-kolibri, @aronasorman got [this error](https://sentry.io/organizations/learningequality/issues/1071646588/?project=1378215&query=is%3Aunresolved).

It turned out that the home folder was missing, and no `kolibri_settings.json` file was available

### Expected behavior

Probably, Kolibri should exit with a helpful error message if there is no home folder or if no settings can be read.

### User-facing consequences

Unhelpful error messages in somewhat common failure scenarios

### Errors and logs

```
IndexError: list index out of range
  File "django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "django/core/handlers/base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "django/core/handlers/base.py", line 215, in _get_response
    response = response.render()
  File "django/template/response.py", line 107, in render
    self.content = self.rendered_content
  File "django/template/response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "django/template/backends/django.py", line 66, in render
    return self.template.render(context)
  File "django/template/base.py", line 207, in render
    return self._render(context)
  File "newrelic/api/function_trace.py", line 121, in dynamic_wrapper
    return wrapped(*args, **kwargs)
  File "django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "django/template/loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "newrelic/api/function_trace.py", line 121, in dynamic_wrapper
    return wrapped(*args, **kwargs)
  File "django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "django/template/library.py", line 203, in render
    output = self.func(*resolved_args, **resolved_kwargs)
  File "kolibri/core/templatetags/kolibri_tags.py", line 136, in kolibri_theme
    return mark_safe(template.format(theme=json.dumps(ThemeHook().theme)))
  File "kolibri/plugins/hooks.py", line 220, in inner
    return func(instance, *args, **kwargs)
  File "kolibri/core/theme_hook.py", line 135, in theme
    theme = list(self.registered_hooks)[0].theme
```

### Steps to reproduce

…

### Context

…

### PR DESCRIPTION
**Pull request by rtibbles**
## Summary
* Adds `setup` command under the `configure` command group
* Allows running setup without running kolibri

## References
Fixes #5669

## Reviewer guidance
Does `kolibri configure setup` work? Is it the right command?

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [x] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/utils/cli.py [hunk 1]
```diff
@@ -491,3 +491,11 @@ def _get_env_vars():
...
 def list_env():
     click.echo_via_pager(_get_env_vars())
+
+
+@configure.command(cls=KolibriDjangoCommand, help="Setup Kolibri")
+def setup():
+    """
+    Setup Kolibri.
+    """
+    logger.info("Kolibri has successfully been setup")
```
Comment (indirectlylit) [inline kolibri/utils/cli.py:501]: Looking at the source of `KolibriDjangoCommand`, it's not clear how or why this works:

https://github.com/learningequality/kolibri/blob/366b49c42652a248fcf0057c32a810bac4a8d392/kolibri/utils/cli.py#L169-L194

If `KolibriDjangoCommand` is implicitly initializing home folders in the background, this information should be added to the docstring.

**Review by indirectlylit — APPROVED** [review]
Initialized home folder

File: kolibri/utils/cli.py (hunk 1 above)
Comment (indirectlylit) [reply kolibri/utils/cli.py:501]: And reading it closer, it says "calls the initialize function"

🤦

**Comment by indirectlylit** [comment]
cc @radinamatic

**Comment by radinamatic** [comment]
Thank you! Can you briefly explain what it the intended use case for this command?

**Comment by rtibbles** [comment]
Setting up the kolibri home directory with all the required files needed for Kolibri to run (which means you can explicitly run this first, before running `kolibri start` or any other command). This allows any upgrade or initialization to happen that might otherwise take some time to be done explicitly beforehand, and let the regular commands be reliably quick!

**Comment by radinamatic** [comment]
Ok, so definitely an _advanced_ management command! 
Will think of best place to add it to 0.15 docs update 👍🏽

**Pull request merged**